### PR TITLE
Initial Viewer CLI

### PR DESCRIPTION
**Pull request by kckst8**
To test:

1.  From a folder outside of your repo, execute "node path/to/viewer/packages/modules/create-itwin-viewer/bin
2. Follow the prompts

When you run the application, you will receive a runtime error about a dompurify import that should be resolved in [this PR ](https://github.com/iTwin/itwinjs-core/pull/3383)

Still need to implement the uiConfig, the desktop template, and some other items. Requesting comments in the meantime

### CODE REVIEW

File: packages/modules/create-itwin-viewer/bin.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+    {
+      type: "text",
+      name: "iTwinId",
+      message: "Enter a default iTwinId",
+    },
+    {
+      type: "text",
+      name: "iModelId",
+      message:
+        "Enter a default iModelId (that is a part of the iTwin that you entered in the previous step)",
```
Comment (johnnyd710) [inline packages/modules/create-itwin-viewer/bin.js:117]: Maybe change to "(_associated with the iTwinId_ that you entered in the previous step)" or similar?

File: packages/modules/create-itwin-viewer/bin.js [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+  const npmMsg = "Installing dependencies...";
+  console.log(npmMsg);
+  process.chdir(appRoot);
+  exec("npm install", () => {
+    finalize(appName);
+  });
+}
+
+function copyTemplate(appRoot, platform) {
+  const generatorRoot = new URL(".", import.meta.url).pathname;
```
Comment (johnnyd710) [inline packages/modules/create-itwin-viewer/bin.js:41]: I believe this line causes an issue on Windows (unless its just my configuration). 
E.g., at this point in my test run, the appRoot is `"C:\\Users\\John.DiMatteo\\repos\\test"` (OK).  
But generatorRoot is `"/C:/Users/John.DiMatteo/repos/viewer.cli/packages/modules/create-itwin-viewer/"` (not OK, beginning forward slash is invalid).
And then templateRoot becomes: `"C:\\C:\\Users\\John.DiMatteo\\repos\\viewer.cli\\packages\\modules\\create-itwin-viewer\\templates\\web"` (ERROR path not found).

Comment (kckst8) [reply packages/modules/create-itwin-viewer/bin.js]: this should be resolved

Comment (johnnyd710) [reply packages/modules/create-itwin-viewer/bin.js]: Just made [a commit](https://github.com/iTwin/viewer/pull/155/commits/a7b338c57ada8c76fe6ed24c93343178f21f4ae3) which should finally fix the issue - just double check I didn't break it on your end @kckst8 and you can close this

**Pull request closed without merging**